[C4GT] Statement: Added tests for StatementRevoked

shashank-iitbhu opened the pull request:
added test `trying_to_update_or_revoke_a_revoked_statement_should_fail`
- Verifies that a revoked statement cannot be updated.
- Ensures that a revoked statement cannot be revoked again.
- Checks for appropriate error handling when attempting invalid operations on a revoked statement.

Fixes: #314

shashank-iitbhu committed:
test added for StatementRevoked

Signed-off-by: Shashank Mittal <shashank.mittal.mec22@itbhu.ac.in>

vatsa287 reviewed (changes requested):
Add similar test for `add_presentation` method too.

shashank-iitbhu committed:
append test for add_presentation

Signed-off-by: Shashank Mittal <shashank.mittal.mec22@itbhu.ac.in>

shashank-iitbhu commented:
> Add similar test for `add_presentation` method too.

Hey @vatsa287 , added test for `add_presentation` in the same test which is now renamed to `trying_to_update_or_revoke_or_add_presentation_for_revoked_statement_should_fail`.
Please review.

shashank-iitbhu committed:
conflicts resolved

Signed-off-by: Shashank Mittal <shashank.mittal.mec22@itbhu.ac.in>

vatsa287 reviewed (approved):
LGTM
